src/components/ui/Tooltip/tests/Tooltip.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Tooltip from '../Tooltip';
+
+describe('Tooltip', () => {
```

Comment:
> test for weird cases when say label is a number, boolean etc
Is this just because this is a public library and could be used in someone's JS project?

Reply:
Ah then should I just be testing that the component renders the toooltip without any errors when non-string props are passed in?